contracts/cosmwasm-vm/cw-icon-light-client/src/state.rs

```diff
@@ -1 +1,544 @@
+use std::cell::RefCell;
 
+use common::icon::icon::lightclient::v1::ClientState;
+use common::icon::icon::lightclient::v1::ConsensusState;
+use common::utils::keccak256;
+use cosmwasm_std::DepsMut;
+use cosmwasm_std::Env;
+use cosmwasm_std::Storage;
+use cw_storage_plus::{Item, Map};
+use prost::Message;
+
+use crate::traits::AnyTypes;
+use crate::traits::Config;
+use crate::traits::IContext;
+use crate::ContractError;
+type ClientId = String;
+const CLIENT_STATES: Map<String, ClientState> = Map::new("CLIENT_STATES");
+const CONSENSUS_STATES: Map<(ClientId, u64), ConsensusState> = Map::new("CONSENSUS_STATES");
+const PROCESSED_TIMES: Map<(ClientId, u64), u64> = Map::new("PROCESSED_TIMES");
+const PROCESSED_HEIGHTS: Map<(ClientId, u64), u64> = Map::new("PROCESSED_HEIGHTS");
+
+const CONFIG: Item<Config> = Item::new("CONFIG");
+
```

Comment:
Instead we can use a sturct to maintain the storage for the lightclient

Reply:
@shreyasbhat0  ya but i thought having them as consts would be cheaper performance wise.